releasenotes-dca/notes/collect-k8s-namespaces-332ee0d0625f4791.yaml

```diff
@@ -0,0 +1,10 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - Collect K8S namespaces
```

Comment:
We tend to categorize new resource collection as a new feature (also really minor but it's maybe clearer for the readers to not use short names in the changelog _Kubernetes_ vs _K8s_)

Reply:
this is good context, thanks